spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/web/WebConfiguration.java

```diff
@@ -124,9 +124,7 @@ public ObjectMapper objectMapper() {
 	}
 
 	@Bean
-	public HttpMessageConverters messageConverters() {
-		final ObjectMapper objectMapper = new ObjectMapper();
-		setupObjectMapper(objectMapper);
+	public HttpMessageConverters messageConverters(ObjectMapper objectMapper) {
```

Comment:
Ideally, we shouldn't be using the explicit HttpMessageConverters like this. Will create an issue to tackle it post RC1.